package.json

```diff
@@ -800,6 +800,19 @@
           "description": "Environment variables that will passed to the processes that run the Go tools (e.g. CGO_CFLAGS)",
           "scope": "resource"
         },
+        "go.gocodeFlags": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "default": [
+            "-builtin",
+            "-ignore-case",
+            "-unimported-packages"
```

Comment:
`unimported-packages` only works for standard packages, correct?

Reply:
Yep, but it's an open to add support for other packages. Hopefully I'll have a chance to add that soon.

Reply:
When 2 packages have the same name (think of text/template, html/template), does `template.` give results from both packages?

Reply:
One package is prioritized and the results for it are returned - so in this case `html/template` is used instead of `text/template` (https://github.com/mdempsky/gocode/blob/master/internal/suggest/stdlib.go).